index.js

```diff
@@ -7,12 +7,105 @@ function ask(questionText) {
   });
 }
 
+
 start();
 
 async function start() {
-  console.log("Let's play a game where you (human) make up a number and I (computer) try to guess it.")
-  let secretNumber = await ask("What is your secret number?\nI won't peek, I promise...\n");
+  console.log("\nLet's play a game where you make up a number and I try to guess it.")
+  let secretNumber = await ask("\nWhat is your secret number?\nI won't peek, I promise...\n");
```

Comment:
Check what happens when the user enters some invalid input for the secretNumber, like the string 'hello' for example.